utils/process-stats-dir.py

```diff
@@ -674,31 +700,8 @@ def main():
                         help="stats-dirs to process")
 
     args = parser.parse_args()
-    if len(args.remainder) == 0:
```

Comment:
Thanks for taking the time to factor this in a separate function!
This makes the intent of the below change easy to grasp.